asyncapi/json-schemas/operational/assignment/attempt/request/assignment-attempt-request.md

```diff
@@ -143,7 +153,7 @@ Remaining not serviced stops/journeys in the assignment will not be serviced (by
 Same as `CANCELLED`
 #### Additional Sign Off codes used by RUTER
 - `EXPIRED`
-  Given a successfull signOn, if the vehicle is not signed off before two hours after the last planned arrival. The vehicle will be automatically signed off with the code `EXPIRED`.
+  Given a successful signOn, if the vehicle is not signed off before two hours after the last planned arrival. The vehicle will be automatically signed off with the code `EXPIRED`.
```

Comment:
Should we precise that this case can be violation of SLA?

Reply:
I think SLA should be kept/retained in the SLA-documents. I feel that changing SLA should not affect the contract...